src/en/help-azure.md

```diff
@@ -179,12 +187,13 @@ You can now run the interactive command:
 juju add-credential azure
 ```
 
-Which will ask for a credential name, and then the values discovered above.
+Which will ask for a credential name, which you determine for yourself, and
+then the values discovered above.
```

Comment:
The phrase "which  you determine for yourself" is vague. It can be construed as yet another variable which you need to find out, but on your own. What about "an arbitrary name" or similar?

We should also list the entities it will ask for:

APP_ID
SUB_ID
TENANT_ID
APP_PASSWORD